src/CloudSecrets/SecretsAnalyzer.cs

```diff
@@ -137,8 +137,9 @@ public bool IsAnalysisSupported(IEnumerable<AnalysisLanguage> languages)
 
         private bool IsRuleActive(string ruleKey)
         {
+            var rulesSettings = ruleSettingsProvider.Get();
             // User settings override the default, if present
-            if (userSettingsProvider.UserSettings.RulesSettings.Rules.TryGetValue(ruleKey, out var ruleConfig))
+            if (rulesSettings.Rules.TryGetValue(ruleKey, out var ruleConfig))
```

Comment:
Is the comment in line 141 still required? It's a bit misleading since we're not just looking at the user settings any more.

Reply:
Ah true, missed that!